metadata/location/buildings.md

```diff
@@ -0,0 +1,72 @@
+# Title
+
+Utah Buildings
+
+## ID
+
+770a9559-3475-4683-80c0-9dfe1cde94e8
+
+## Brief Summary
+
+Polygon dataset of building footprints in Utah derived from Microsoft.
+
+## Summary
+
+This dataset contains polygons representing the approximate footprint of buildings in Utah. UGRC derived these footprints from Microsoft and made adjustments to provide a Utah-focused dataset that minimizes errors where possible. This dataset contains building polygon geometries only and does not show the ownership or use of the buildings.
+
+## Description
+
+### What is the dataset?
+
+A building footprint is a two-dimensional outline of a three-dimensional, real-world building. These footprints are often digitally represented as polygons and can be used in a number of applications, including city and county planning, construction management, and demographic analysis. This dataset contains building footprints as polygons for physical structures found within Utah.
+
+### What is the purpose of the dataset?
+
+This dataset has been made available to the public for general reference and analytic purposes.
+
+<!--- Do we, or any other state agency, use this dataset for anything more specific? --->
```

Comment:
I'll try again with the I think these are in the base maps statement again. Maybe Zach will agree with me this time haha.

Reply:
Yep, building footprints are in Lite and Terrain.